cwltool/pathmapper.py

```diff
@@ -191,6 +192,17 @@ def files(self) -> List[str]:
     def items(self) -> List[Tuple[str, MapperEnt]]:
         return list(self._pathmap.items())
 
+    def items_exclude_children(self) -> List[Tuple[str, MapperEnt]]:
+        newitems = {}
+        keys = [key for key, entry in self.items()]
+        for key, entry in self.items():
+            if entry.type in ("File", "Directory"):
```

Comment:
@jfennick I think this line can be removed. Debugging the conformance test that @mr-c pointed is failing, I noticed that the `File` to be staged by the workflow is a `CreateFile` (which I was not familiar with, but the code comments help clarify it, I think - <https://github.com/common-workflow-language/cwltool/blob/3945a18c699c3ca500a9343c40b34db966c6787b/cwltool/pathmapper.py#L48-L49>).

After I removed that line, the test `cat-from-dir.cwl` passed, and the workflow `test.cwl` in #1029 also passed with no errors (I switched to `main` and confirmed it failed for me too there).